lua/entities/ent_jack_turret_base/init.lua

```diff
@@ -616,10 +646,13 @@ function ENT:Traverse()
     end
 
     if IsValid( self.flashlight ) then
-        self.flashlight:SetLocalAngles( self:WorldToLocalAngles( self:GetAttachment( 1 ).Ang ) )
+        self.flashlight:SetLocalAngles( Angle( self.CurrentSwing, self.CurrentSweep, 0 ) )
     end
 end
 
+function ENT:AdditionalShootFX()
+end

```

Comment:
Can you add a comment explaining why this was emptied?

Reply:
Done!